src/plugins/vis_type_timeseries/server/lib/vis_data/helpers/get_bucket_size.ts

```diff
@@ -76,24 +72,24 @@ export const getBucketSize = (
   capabilities: SearchCapabilities,
   bars: number
 ) => {
-  const defaultBucketSize = calculateBucketSizeForAutoInterval(req, bars);
-  let intervalString = `${defaultBucketSize}s`;
```

Comment:
It looks like you're no longer setting the initial value here. What side effects will this have?

Reply:
I've renamed calculateBucketSizeForAutoInterval -> calcAutoInterval cause in fact it's not a bucket size  and it confused me a little.  About initial value for interval string I think we need it only for `auto` mode,  I just move that to end of that method. But if you see that I broke something, please let me know.